### DISCUSSION TITLE
My engineering thesis: Development of controlled and trackable ultrasound probe system

### DISCUSSION THREAD

**Discussion by martynakielbasa** [PlusToolkit/PlusLib#1050] [Show and tell]
Hello! 

I'm more than happy to announce that I passed my diploma exam a few days ago at Wroclaw University of Science and Technology, in Poland! In my research I used PLUS for probe calibration and 3D Slicer for live volume reconstruction. I also used some printable 3D models provided by you. I naturally cited all of your works to give you all the credit you deserve. 

The main aim of my work was to develop a tracked ultrasound probe system used in the process of imaging human bone structures. The measuring system consisted of NDI Polaris Spectra navigation system and Telemed EchoBlaster ultrasound. 

My calibration errors were as following:

- 0.28 - stylus calibration, 
- 0.50 - phantom registration, 
- 0.34 - probe calibration. 

This is what a reconstructed model of femur phantom looked like: 

![image](https://user-images.githubusercontent.com/111685804/217622889-e076064b-9b13-4dad-9b43-30227d03471c.png)

I took some measurements and compared the model reconstructed with my system to a model of the same bone reconstructed using CT. The volume reconstruction errors ranged from 0.79% to 3.08%.

Overall, my thesis was a part of a bigger project. Its goal was to create an ultrasound tomography system. The ultrasound tomography system in still under construction - however I'm planning to run some tests in the future. 

I would like to thank all the people here who helped me during the process, especially @Sunderlandkyl. I'm really grateful for all the support and advice from you! 

Best regards, 
Martyna

**Comment by ungi** [comment]
Sounds awesome! Could you please put a link here to some of your publications. And maybe your thesis if it's public. Thanks!

**Comment by lassoan** [comment]
Very nice. Thanks a lot for sharing. I hope you remain in the Slicer/Plus/SlicerIGT community. There are many interesting projects to work on and companies and research groups are always looking for good hires.

**Comment by Sunderlandkyl** [comment]
Awesome, I'm glad to have been of help! I would also be interested in reading your publications/thesis if they are publicly available!
